index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Evans_Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="navigation">
+      <div class="logo">
+        <h3>Evans</h3>
+      </div>
+      <nav class="navlinks">
+        <ul class="links">
+          <li class="link"><a href="#portfolio">Portfolio</a></li>
+          <li class="link"><a href="#about">About</a></li>
+          <li class="link"><a href="#contact">Contact</a></li>
+          <li>
+            <a href="#"><i class="fa-regular fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+
+    <section id="hero">
+      <div class="hero_text">
+        <h1 class="title">
+          Hey There <br />
```

Comment:
- [ ] Kindly ensure that you follow suit with all the design guidelines to achieve an optimized pixel-perfect design like you'll be doing in your next job. Note that everything must much as possible, the font-size, font-weight, margin, padding, background-image, color, line-heigth, font-family and the likes.

- Your design
<img width="183" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227179766-dce76025-e788-42a9-836d-317368386968.png">

- Figma design
<img width="217" alt="right" src="https://user-images.githubusercontent.com/41163571/227010023-9fcfb014-309c-4293-ba46-014f8e1c0000.png">

I took the extra mile for your sake to confirm if the font existed and was for free, guess what? 
[it is!!!!!!!](https://fonts.google.com/specimen/Crete+Round?query=crete)